### DISCUSSION TITLE
Feature Request: Custom Shortcuts Quick Info Popover

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [rxhanson/RectanglePro-Community#292] [Ideas]
Rectangle Pro is awesome. I feel like I have handicap using macOS without it installed.

And yet, the only thing I use it for at all us Custom Shortcuts. What an epic feature. The bit I specifically depend on is the ability to different behavior when pressing the key again. Here is how I use it:

![image](https://github.com/rxhanson/RectanglePro-Community/assets/26592486/7fa2ed21-e1b5-41f4-b3ea-ba70154d78f6)

Here's the thing though: As I cycle through the options I've defined—particularly when hitting the Left or Right Arrow (as you can see, there are 6 cycling options defined) it would be super nice if I could have a "popover" box show up for a second on the screen letting me know what dimensions I just set (perhaps with a custom message). For example, on the first Left Arrow press, it would say: "Left 4/5".

**Comment by rxhanson** [comment]
Thanks for the feedback. This is the first I've heard this request, and perhaps your use case of having so many repetitions is slightly unusual. Personally, I've always found popup messages really distracting/annoying, and that's why I typically avoid them. I'm trying to think if there's some sort of underlying issue here that would be more elegant to solve. One idea is just having something less intrusive, like ruler markers that show on the edge of the display when you execute a shortcut.

**Reply by AaronBeaudoin** [reply]
That sounds like a beautiful solution, but how would you decide what "ticks" to show along the "ruler"? Would it:
- Show ticks based on the "steps" defined in the Custom Shortcut you just executed?
- Allow you to customize the ticks you want to show when executing each Custom Shortcut?
- Show some standard set at all times?

In any case, the main goal here it to make it clear what dimensions I'm setting every time I press my Custom Shortcut. Sometimes it's tough to "eyeball" what the current fractional dimensions of the window are.
